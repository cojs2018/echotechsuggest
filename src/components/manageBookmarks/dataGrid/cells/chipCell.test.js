import React from 'react';
import render from 'react-test-renderer';

import ChipCell from "./chipCell";

describe('Cell', () => {
    it('renders correctly', () => {
        const props = {
            values: ['tag0', 'tag1', 'tag2']
        }

        const tree = render
            .create(<ChipCell {...props} />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
})