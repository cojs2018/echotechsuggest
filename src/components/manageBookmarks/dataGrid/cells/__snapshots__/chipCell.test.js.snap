// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      false,
      Array [
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
        },
        undefined,
        undefined,
      ],
    ]
  }
>
  <Text
    numberOfLines={1}
    style={
      Array [
        Object {
          "color": "#000000",
          "fontFamily": "System",
          "fontWeight": "400",
          "textAlign": "left",
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#ebebeb",
          "borderColor": "#ebebeb",
          "borderRadius": 16,
          "borderStyle": "solid",
          "borderWidth": 0.5,
          "elevation": 0,
          "flexDirection": "row",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": false,
            "selected": false,
          }
        }
        accessible={true}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 16,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "paddingLeft": 4,
                "position": "relative",
              },
              Object {
                "paddingRight": 4,
              },
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "lineHeight": 24,
                    "marginVertical": 4,
                    "minHeight": 24,
                    "textAlignVertical": "center",
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "System",
                    "fontWeight": "400",
                    "marginLeft": 8,
                    "marginRight": 8,
                  },
                  undefined,
                ],
              ]
            }
          >
            tag0
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#ebebeb",
          "borderColor": "#ebebeb",
          "borderRadius": 16,
          "borderStyle": "solid",
          "borderWidth": 0.5,
          "elevation": 0,
          "flexDirection": "row",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": false,
            "selected": false,
          }
        }
        accessible={true}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 16,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "paddingLeft": 4,
                "position": "relative",
              },
              Object {
                "paddingRight": 4,
              },
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "lineHeight": 24,
                    "marginVertical": 4,
                    "minHeight": 24,
                    "textAlignVertical": "center",
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "System",
                    "fontWeight": "400",
                    "marginLeft": 8,
                    "marginRight": 8,
                  },
                  undefined,
                ],
              ]
            }
          >
            tag1
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#ebebeb",
          "borderColor": "#ebebeb",
          "borderRadius": 16,
          "borderStyle": "solid",
          "borderWidth": 0.5,
          "elevation": 0,
          "flexDirection": "row",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": false,
            "selected": false,
          }
        }
        accessible={true}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 16,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "paddingLeft": 4,
                "position": "relative",
              },
              Object {
                "paddingRight": 4,
              },
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "lineHeight": 24,
                    "marginVertical": 4,
                    "minHeight": 24,
                    "textAlignVertical": "center",
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "System",
                    "fontWeight": "400",
                    "marginLeft": 8,
                    "marginRight": 8,
                  },
                  undefined,
                ],
              ]
            }
          >
            tag2
          </Text>
        </View>
      </View>
    </View>
  </Text>
</View>
`;
